integration/smi_test.go

```diff
@@ -177,3 +186,10 @@ func (s *SMISuite) checkTCPServiceServerURLs(c *check.C, config *dynamic.Configu
 		c.Log("Service " + name + " has the correct expected values.")
 	}
 }
+
+func (s *SMISuite) waitForPodIPs(c *check.C, pods []string) {
+	for _, pod := range pods {
+		c.Log("Waiting for pod: \"" + pod + "\" to have IP assigned.")
```

Comment:
Apparently, `c.Log` relies on `Sprintf` to assemble arguments. It gives us the ability to use the `%q`:
```suggestion
		c.Log("Waiting for pod: %q to have IP assigned.", pod)
```